tools/riassigner/riassigner_from_comment.xml

```diff
@@ -15,7 +15,11 @@
 
     <command detect_errors="exit_code"><![CDATA[
```

Comment:
I have the feeling that the result writing could be simplified here, to avoid having to use `mv` etc.

Reply:
I haven't really changed the command much as It is a bit tricky because the writer expects the filename with an extension, so I can't pass the tool output directly to it. Also cannot use the from_work_dir because we have multiple extensions.